index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+</head>
+<body>
+ 
+<header class="navbar">
+    <div class="logo"><a href="#"></a>Eros</div>
+    <div class="menu"><i class='bx bx-menu'></i></box-icon></div>
+    
+</header>
+<section> 
```

Comment:
- [ ] Good job so far, I noticed that you forget to add your social media account, as required for this project the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag), kindly add this section , I also encourage you to follow all requirements for this project :wink: